File: python/src/commands/GetGreetingCommand.py
from typing import List
from src.commands import ICommand
from src.services import GreetingService


class GetGreetingCommand(ICommand):
    def __init__(self, greetingService: GreetingService) -> None:
        self._greetingService = greetingService

    def invoke(self, tokens: List[str]) -> None:
        id = int(tokens[1])
        try:
            g = self._greetingService.getGreeting(id)
            print(g)
        except Exception as e:
            print(e)

File: java/src/main/java/com/example/demo/commands/GetGreetingCommand.java
package com.example.demo.commands;

import java.util.List;

import com.example.demo.entities.Greeting;
import com.example.demo.services.GreetingService;

public class GetGreetingCommand implements ICommand{
    
    private final GreetingService greetingService; 

    public GetGreetingCommand(GreetingService greetingService) {
        this.greetingService = greetingService;
    }

    @Override
    public void invoke(List<String> tokens) {
        Long id = Long.parseLong(tokens.get(1));
        try {
            Greeting getGreet = greetingService.getGreeting(id);
            System.out.println(getGreet);
        } catch (Exception e){
            System.out.println(e.getMessage());
        }
    }
}
